umm-spec-lib/src/cmr/umm_spec/umm_to_xml_mappings/echo10.clj

```diff
@@ -187,10 +187,13 @@
                  ;; expectes a string in %9.2f format, so we have to
                  ;;ignore 'Free', 'Gratis', etc.
                  (catch NumberFormatException e)))]
-     [:DataFormat (-> c
-                      :ArchiveAndDistributionInformation
-                      :FileDistributionInformation
-                      first :Format)]
+     ; Go through all of the FileDistributionInformation elements and set the
+     ; ECHO 10 DataFormats.
+     (let [data-formats (-> c
+                            :ArchiveAndDistributionInformation
+                            :FileDistributionInformation)]
+       (for [data-format data-formats]
```

Comment:
Alternative to the `for`:

`[:DataFormat (map :Format data-formats)]`

Reply:
thanks